app.js

```diff
@@ -16,7 +15,11 @@ import {
   oauthLoginAuthorize,
   oauthLogoutAuthorize,
 } from './controllers/oauthAuthentication';
+import {
+  getLogin,
+} from './controllers/loginController';
 import { oauthDataCallback, getData, oauthDataAuthorize } from './controllers/oauthData';
+import validateLoginCallback from './validators/loginCallbackValidator';
```

Comment:
validateLoginCallback => in my opinion this does not describe well what this function does. 

What about "validateCallbackParamsMiddleware" ?

Reply:
No prob, I have no advice on that